ccp/src/cu/proving_thread/async_/thread.rs

```diff
@@ -166,3 +172,20 @@ impl ProvingThreadFacade for ProvingThreadAsync {
         Ok(())
     }
 }
+
+#[derive(Debug, Clone)]
```

Comment:
let's add `Copy` here

Reply:
Well, we may add something like string to the fields, and it will not become Copy anymore.  I.e. it is not Copy by it nature.